README.md

```diff
@@ -39,6 +39,11 @@ Our [contribution guide](https://github.com/wildfly-security/wildfly-elytron/blo
  
 For more details, check out our [getting started guide](https://wildfly-security.github.io/wildfly-elytron/getting-started-for-developers/) for developers.
 
+Example Feature demos
```

Comment:
Thanks @Akshayabhat, just a small nitpick :) Could we substitute ```demos``` for ```Demos``` to stay consistent with the document capitalization.

Reply:
yep, good catch! Updated!